hypothesis-python/src/hypothesis/strategies.py

```diff
@@ -45,6 +45,11 @@
     check_valid_sizes, check_valid_integer, check_valid_interval, \
     check_valid_magnitude
 
+if False:
+    from typing import Any, Dict, Union, Sequence, Callable, Pattern  # noqa
+    # See https://github.com/python/mypy/issues/3186 - numbers.Real is wrong!
+    Real = Union[int, float, Fraction, Decimal]
```

Comment:
😭

Reply:
There *is* an issue open to use structural rather than nominal typing for this, which I can probably implement later (as I did for regex constants last time).  Even then, Decimal isn't an instance of Real at runtime and probably won't be for Mypy either :sob: